libraries/core/classes/AppCommand/index.js

```diff
@@ -1,169 +1,171 @@
 /* global SGJavascriptBridge */
-import fetch from 'isomorphic-fetch';
-import event from '../Event';
 import { logger, hasSGJavaScriptBridge } from '../../helpers';
+import { isValidVersion, getLibVersion, isVersionAtLeast } from '../../helpers/version';
 import logGroup from '../../helpers/logGroup';
+import DevServerBridge from '../DevServerBridge';
 
 /**
  * The app command class.
  */
 class AppCommand {
   /**
    * @param {boolean} log Whether the command will be logged.
+   * @param {boolean} checkLibVersion Whether the lib version will be checked before dispatch.
    */
-  constructor(log = true) {
+  constructor(log = true, checkLibVersion = true) {
     this.log = log;
-    this.command = {};
+    this.checkLibVersion = checkLibVersion;
+    this.name = '';
+    this.params = null;
     this.libVersion = '9.0';
+    this.commandsWithoutLog = [
+      'sendPipelineRequest',
+      'sendHttpRequest',
+      'getWebStorageEntry',
+    ];
   }
 
   /**
    * Sets the command name.
    * @param {string} name The command name.
-   * @returns {AppCommand}
+   * @return {AppCommand}
    */
   setCommandName(name) {
-    this.command.c = name;
+    if (typeof name === 'string') {
```

Comment:
Please add a guard clause for better code quality and readability instead.

Reply:
The concept of guard clauses is to return early in cases where the return value of a method can differ due to the occurrence of various conditions.

In a case where the return value is initialized with a default value and then updated within a complex if else tree i would totally accept your complain, but here the method only returns `this`.

With a guard clause it would need to return `this` in two situations which reduces the readability from my point of view. So i'd be happy if i could keep the code like it is within this and the other two mentioned methods.